BraveShared/BraveStrings.swift

```diff
@@ -297,6 +297,58 @@ public extension Strings {
     public static let ErrorPagesVisitOnceButton = NSLocalizedString("ErrorPagesVisitOnceButton", tableName: "BraveShared", value: "Visit site anyway", comment: "Button label to temporarily continue to the site from the certificate error page")
 }
 
+// MARK: - Sync
+public extension Strings {
```

Comment:
Please add `value` to all of these strings. For now, the key can just be the variable name. For example:

public static let QRCode = NSLocalizedString("QRCode, value: "QR Code", comment: "QR Code section title")